src/test/java/it/pagopa/pn/user/attributes/rest/v1/ConsentsControllerTest.java

```diff
@@ -21,7 +22,7 @@
     @Autowired
     WebTestClient webTestClient;
 
-    @Test
```

Comment:
La build fallisce poiché la code coverage è inferiore alla soglia minima impostata. Per realizzare Unit Test che utilizzano Dynamo abbiamo implementato dei Mock del DB che permettono l'inserimento ed il recupero degli Items.
Un esempio può essere trovato qui: [TimelineDaoDynamoTest](https://github.com/pagopa/pn-delivery-push/blob/develop/src/test/java/it/pagopa/pn/deliverypush/middleware/timelinedao/TimelineDaoDynamoTest.java)

Reply:
Aggiunti test